packages/core/docs/general/key-concepts.md

```diff
@@ -1,32 +1,32 @@
+This document will walk you through the most important concepts of Vue Storefront. Once you'll grab the ideas behind the software it should be fairly straightforward for you to use it in the right way.
 
 ## Configuration
 
-The first thing you usually want to do after setting up a new project is some configuration. No matter if you want to change your backend API credentials, change routes or add a custom logger there is always a single place to do all these things - Vue Storefront Modules configuration in `nuxt.config.js`.
+The first thing you usually want to do after setting up a new project is some configuration. No matter if you want to change your backend API credentials, change routes or add a custom logger, there is always a single place to do all these things - Vue Storefront Modules configuration in `nuxt.config.js`.
 
-If you're using our boilerplate you will find 3 Vue Storefront modules in your configuration:
-- `@vue-storefront/nuxt` - Our core Nuxt module. It's main responsibility is to extend Nuxt configuration. 
+If you're using our boilerplate, you will find 3 Vue Storefront modules in your configuration:
+
+- `@vue-storefront/nuxt` - Our core Nuxt module. Its main responsibility is to extend Nuxt configuration.
 - `@vue-storefront/nuxt-theme`- This module adds routing and theme-specific configuration for Nuxt.
-- `@vue-storefront/<ECOMMERCE_PLATFORM>` - This is a module of your eCommerce integration. All configuration related to specific eCommerce platform like setting up API credentials has to happen through this module. Such module usually provides some additional functionalities like setting up the cookies.
+- `@vue-storefront/<ECOMMERCE_PLATFORM>` - This is a module of your eCommerce integration. All configuration related to the specific eCommerce platform like setting up API credentials has to happen through this module. Such module usually provides some additional functionalities like setting up the cookies.
 
 ## Composables
 
 ::: tip Composables? Is this a French meal?
-Composable is a function that uses Vue.js Composition API under the hood. It's commonly named with a `use` prefix (eg. `useProduct`, `useCart`). This convention comes from the React community where we can find a very similar pattern - Hooks which inspired Vue.js core team to introduce the Composition API.
+Composable is a function that uses [Vue.js Composition API](https://v3.vuejs.org/guide/composition-api-introduction.html) under the hood. It's commonly named with a `use` prefix (eg. `useProduct`, `useCart`). This convention comes from the React community where we can find a very similar pattern - [Hooks](https://reactjs.org/docs/hooks-intro.html), which inspired Vue.js core team to introduce the Composition API. Don't worry if you never worked with this API, you don't need a previous experience to start using VSF composables
```

Comment:
**note**: maybe I am overpromising here but it's my impression so far 🤔 please correct me if I am wrong

Reply:
I think it is needed to know CAPI